python/triton_kernels/tests/test_matmul.py

```diff
@@ -402,6 +400,9 @@ def _test_op(m, n, k, split_k, do_gather, do_scatter, inner_expt_opt, do_gamma,
                            gather_indx, scatter_indx, precision_opt,
                            gammas=gammas, epilogue=epilogue, c=c,
                            fused_activation=fused_activation)
+        # Apply output transpose if needed
+        if c_transpose:
+            tri_y = tri_y.mT.contiguous().mT
```

Comment:
This defeats the purpose of `c_transpose`, which is to test the kernel with an output parameter that is transposed.